src/Rector/Neon/RenameMethodNeonRector.php

```diff
@@ -66,7 +72,11 @@ public function configure(array $configuration): void
 
     public function changeContent(string $content): string
     {
-        foreach ($this->methodCallRenames as $methodCallRename) {
+        $methodCallRenames = array_merge(
+            $this->methodCallRenameCollector->getMethodCallRenames(),
+            $this->methodCallRenames
+        );
+        foreach ($methodCallRenames as $methodCallRename) {
```

Comment:
From name of `methodCallRenameCollector` I assume it contains all the method renames, not just some.

```suggestion
        foreach ($this->methodCallRenameCollector->getMethodCallRenames() as $methodCallRename) {
```

Reply:
Maybe, so you want to completely remove configuration (and methodCallRenames array) from this rector? I thought you want to keep it for some reason :)

Reply:
Not sure what exactly should be done :), but we should avoid this kind of bug.

Reply:
I think there is no reason to have configuration in this rule. It should be shared from RenameMethodRector (via MethodCallRenameCollector). I can't find any use case when I would want to rename different methods in php and different in neon / latte.

Reply:
@TomasVotruba any decision here? :)

Reply:
> I think there is no reason to have configuration in this rule. It should be shared from RenameMethodRector (via MethodCallRenameCollector).

Agreed :+1: 

Do you think it should be bidirectional? For `RenameMethodRector` and the Neon rule?

Reply:
I prefere to configure it only in RenameMethodRector and propagate to others. And it would be awesome if the others will be registered automatically. I saw something like that in config.php:

    $services->set(RenameClassNonPhpRector::class);
};

If no rename rules are configured, nothing will happend. But if there are some renames, Rector will change them in all file types.

Reply:
@lulco I think it cannot be done better. Let's go for it :+1:

Reply:
sorry for delay :) I've rewrite the code like we discussed, please check it again